crates/wash-cli/README.md

```diff
@@ -94,49 +94,56 @@ nix run github:wasmCloud/wash
 
 ### build
 
-Builds and signs the actor, provider, or interface as defined in a `wasmcloud.toml` file.  Will look for configuration file in directory where command is being run.  
+Builds and signs the actor, provider, or interface as defined in a `wasmcloud.toml` file. Will look for configuration file in directory where command is being run.  
 There are three main sections of a `wasmcloud.toml` file: common config, language config, and type config.
 
 #### Common Config
+
 | Setting       | Type   | Default                       | Description                                                                            |
 | ------------- | ------ | ----------------------------- | -------------------------------------------------------------------------------------- |
-| name          | string |                               | Name of the project                                                                    | 
+| name          | string |                               | Name of the project                                                                    |
 | version       | string |                               | Semantic version of the project                                                        |
-| path          | string | `{pwd}`                       | Path to the project directory to determine where built and signed artifacts are output | 
-| wasm_bin_name | string | "name" setting                | Expected name of the wasm module binary that will be generated                         |
+| path          | string | `{pwd}`                       | Path to the project directory to determine where built and signed artifacts are output |
 | language      | enum   | [rust, tinygo]                | Language that actor or provider is written in                                          |
 | type          | enum   | [actor, provider, interface ] | Type of wasmcloud artifact that is being generated                                     |
-
+| wasm_bin_name | string | "name" setting                | Expected name of the wasm module binary that will be generated                         |
```

Comment:
Just a note, since I'm guessing refactoring would be non-trivial, but your comment about moving the build command to actor config made me ask: shouldn't this also be under actor config? I.e. it's not relevant for building providers (until we have WASI providers)

Reply:
Yeah this was a really weird one. It's strange because this isn't used in the project config most of the time, usually it's used in the context of a Rust project that we infer the binary name from the Cargo.toml `[lib]` section. Let me see if I could easily refactor it